### DISCUSSION TITLE
Favicon on the public page for each radio station

### DISCUSSION THREAD

**Discussion by amarildoazura** [AzuraCast/AzuraCast#6888] [General]
Good morning Dear friend of this wonders community, I would like to know if it is possible to have a favicon for each radio within the panel, as I have several radios I would like to use a different favicon for each radio.
And if possible, it can be done through the panel of each radio.

Gratitude for this community

Amarildo Konorat, Joinville, Santa Catarina, Brazil

**Comment by BusterNeece** [comment]
This is not currently a feature in the system, but you're welcome to submit it as a new feature idea on https://features.azuracast.com/

Currently, there may be a pull request coming up to add this functionality soon as well.
